teeproxy.go

```diff
@@ -34,6 +34,8 @@ var (
 	forwardClientIP       = flag.Bool("forward-client-ip", false, "enable forwarding of the client IP to the backend using the 'X-Forwarded-For' and 'Forwarded' headers")
 	closeConnections      = flag.Bool("close-connections", false, "close connections to the clients and backends")
 	logging               = flag.Bool("logging", false, "enable profiling logging")
+	maxIdleConns          = flag.Int("maxIdleConns", 5000, "maximum idle connections")
```

Comment:
what's the reasoning behind this having the same limit as `maxIdleConnsPerHost`? Shouldn't it be higher?